packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.js

```diff
@@ -0,0 +1,37 @@
+import { withThemeContext } from '@zooniverse/react-components'
+import { Button } from 'grommet'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { bool, shape, string } from 'prop-types'
+import React from 'react'
+
+import theme from './theme'
+
+function WorkflowSelectButton (props) {
+  const { workflow } = props
+  const router = useRouter()
+
+  const as = workflow.default
+    ? `${router.asPath}/classify`
+    : `${router.asPath}/classify/workflow/${workflow.id}`
```

Comment:
Woohoo! This is going to make building out workflow selection easier I think having a project home page to actually use for manual testing.

Reply:
I'm actually thinking that all workflows should be routed to `/classify/workflow/${workflow.id}`, and that `/classify` simply does a redirect for consistency. Thoughts / feelings?

Reply:
Yeah, I'm fine with `/classify` redirecting to `/classify/workflow/:workflow_id` each time, but this should be in an ADR (if it's not already).

Reply:
Sounds good to me but `/classify` should redirect to the stored workflow ID from project preferences for logged-in volunteers, rather than just redirecting to the default workflow.

Reply:
https://github.com/zooniverse/front-end-monorepo/issues/806

Reply:
Yeah, the same ordering of selection would exist: user stored UPP workflow id --> project set workflow id in UPP --> project default --> random active selection. Any one of those would result in the id that gets redirected to.

Reply:
also https://github.com/zooniverse/front-end-monorepo/pull/1107

Reply:
Also - for the purposes of this PR, I propose leaving as is for now

Reply:
I think you can get rid of the ternary here and always route to the workflow ID when linking to a specific workflow (assuming these links are the same as the 'Choose a workflow' links on the current home page.)